shopify.com.subdomains.json

```diff
@@ -0,0 +1,19 @@
+{
+  "providerId":"Shopify.com",
+  "providerName":"Shopify",
+  "serviceId":"subdomains",
+  "serviceName":"Shopify Subdomain",
+  "version": 1,
+  "syncPubKeyDomain": "shopify.com",
+  "logoUrl":"https://domainconnect.org/wp-content/uploads/2016/08/shopify.png",
+  "description":"Enables a subdomain to work with Shopify",
+  "variableDescription":"subdomain is the specific subdomain prefix and v1 is the shopify hosted domain name.",
+  "records":[
+     {
+        "type":"CNAME",
+        "host":"%subdomain%",
```

Comment:
Hi, template like this would block the possibility of 2 instances for 2 subdomains (at least for some providers).
Recommended way: host setting = @ for this record and hostRequired=true on the template level.
With this setting "host" parameter would carry the subdomain name.

Reply:
Thanks for the feedback! I made the update, let me know if that looks fine 👍